spec.html

```diff
@@ -903,21 +903,22 @@ <h1>Static Semantics</h1>
-      <p>Conversions between mathematical values and numbers are never implicit, and always explicit in this document. A conversion from a mathematical value to a Number is denoted as "the Number value for _x_", and is defined in <emu-xref href="#sec-ecmascript-language-types-number-type"></emu-xref>. A conversion from a Number to a mathematical value is denoted as "the <dfn id="mathematical-value">mathematical value</dfn> of _x_", or ℝ(_x_). Note that the mathematical value of non-finite values is not defined, and the mathematical value of *+0* and *-0* is the mathematical value 0<sub>ℝ</sub>.</p>
-      <p>When the term <dfn id="integer">integer</dfn> is used in this specification, it refers to a Number value whose mathematical value is in the set of integers, unless otherwise stated: when the term <dfn id="mathematical integer">mathematical integer</dfn> is used in this specification, it refers to a mathematical value which is in the set of integers. As shorthand, integer<sub>_t_</sub> can be used to refer to either of the two, as determined by _t_.</p>
+      <p>In certain contexts, an operation is specified which is generic between Numbers, BigInts and mathematical values. In these cases, the subscript can be a variable; _t_ is often used for this purpose, for example 5<sub>_t_</sub> &times; 10<sub>_t_</sub> = 50<sub>_t_</sub> for any _t_ ranging over ℝ, 𝔽 and ℤ, since the values involved are within the range where the semantics coincide.</p>
+      <p>Conversions between mathematical values and numbers are never implicit, and always explicit in this document. A conversion from a mathematical value to a Number is denoted as "the Number value for _x_" or 𝔽(_x_), and is defined in <emu-xref href="#sec-ecmascript-language-types-number-type"></emu-xref>. A conversion from a Number to a mathematical value is denoted as "the <dfn id="mathematical-value">mathematical value</dfn> of _x_", or ℝ(_x_). Note that the mathematical value of non-finite values is not defined, and the mathematical value of *+0*<sub>𝔽</sub> and *-0*<sub>𝔽</sub> is the mathematical value 0<sub>ℝ</sub>.</p>
+      <p>When the term <dfn id="integer">integer</dfn> is used in this specification, it refers to a mathematical value which is in the set of integers, unless otherwise stated: when the term <dfn id="a Number whose mathematical value is an integer">a Number whose mathematical value is an integer</dfn> is used in this specification, it refers to a Number value which its mathematical value is in the set of integers. As shorthand, integer<sub>_t_</sub> can be used to refer to any of the three, as determined by _t_.</p>
```

Comment:
```suggestion
      <p>When the term <dfn id="integer">integer</dfn> is used in this specification, it refers to a mathematical value which is in the set of integers, unless otherwise stated: when the term <dfn id="a Number whose mathematical value is an integer">a Number whose mathematical value is an integer</dfn> is used in this specification, it refers to a Number value whose mathematical value is in the set of integers. As shorthand, integer<sub>_t_</sub> can be used to refer to any of the three, as determined by _t_.</p>
```

I've been wanting to address the usage of "integer"/"integral" in the spec for a while now, and I think this is an appropriate place to do it. I'd prefer to omit the integer<sub><em>t</em></sub> shorthand and use "integer" only for mathematical values, "integral Number" for Number values whose Mathematical value is an integer, and just BigInt for BigInts.